recipes/rocksdb/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "5.8" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha1: f0a5119bef4e1494e1af3871c288a1fe8200f304
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - snappy
+
+  run:
+    - snappy
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/librocksdb.dylib            # [osx]
+    - test -f ${PREFIX}/lib/librocksdb.so               # [linux]
```

Comment:
Are there any headers we should check for as well? If we add static libraries, we should include them here too.